lms/templates/navigation.html

```diff
@@ -75,6 +76,28 @@ <h2 class="course-header"><span class="provider">${course.display_org_with_defau
       </%block>
     </ol>
     <ol class="user">
+        % if settings.FEATURES.get('SHOW_LANGUAGE_SELECTOR', False):
```

Comment:
It would be nice if we could hide this check behind lang_pref instead of referencing the configuration directly in the template.